strongbox-cron-tasks/src/main/java/org.carlspring.strongbox/cron/config/JobManager.java

```diff
@@ -0,0 +1,31 @@
+package org.carlspring.strongbox.cron.config;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.quartz.SchedulerFactoryBean;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Kate Novik.
+ */
+@Component
+public class JobManager
+        implements IJobManager
+{
+
+    private ConcurrentMap<String, Boolean> executedJobs = new ConcurrentHashMap<>();
```

Comment:
Please, use a generic `Map` here like this:

    private Map<String, Boolean> executedJobs = new ConcurrentHashMap<>();